src/test/java/com/thoughtworks/pacman/core/GameTest.java

```diff
@@ -96,4 +97,11 @@ public void advance_shouldTellPacmanToDie_whenGhostsKillPacman() throws Exceptio
 
         verify(pacman).die();
     }
+
+    @Test
+    public void GameEntryControl() throws Exception {
```

Comment:
metot ismi kodun geneliyle uyumlu değil. `gameEntryControl` olabilir belki. Ama isim biraz manasız geldi. Neyi test ettiği isimden anlaşılmıyor. Test in adı test olursa neyi test ettiği anlaşılmaz. Specific test case neyse onu isimlendir.